tex/latex/fks/problems.tex

```diff
@@ -95,7 +96,10 @@
 průměr~\@probavg;
 \plural{\@probsolvers}{řešil}{řešili}{řešilo}{}~\@probsolvers{}~\plural{\@probsolvers}{student}{studenti}{studentů}}}
```

Comment:
Nebylo to potřeba, tak jsem to z lenostních důvodů nepřekládal. Ale není problém to dopřeložit. Mám to dělat?